src/funding/venmo/config.jsx

```diff
@@ -46,6 +47,10 @@ export function getVenmoConfig() : FundingSourceConfig {
             [ BUTTON_COLOR.GOLD ]:   BUTTON_COLOR.BLUE,
             [ BUTTON_COLOR.BLUE ]:   BUTTON_COLOR.SILVER,
             [ BUTTON_COLOR.SILVER ]: BUTTON_COLOR.BLUE
+        },
+
+        eligible: () => {
```

Comment:
Could we incorporate this check in the `supportedNativeBrowser` prop? 

- https://github.com/paypal/paypal-checkout-components/blob/master/src/zoid/buttons/component.jsx#L506-L510
- https://github.com/paypal/paypal-checkout-components/blob/master/src/zoid/buttons/util.js#L22-L40

Venmo is already using the `supportedNativeBrowser` prop to help determine eligibility.

Reply:
@gregjopa This is a great point!  Taking a look at it.